_distro_map.yml

```diff
@@ -240,6 +246,9 @@ openshift-telco:
     enterprise-4.14:
       name: '4.14'
       dir: container-platform-telco/4.14
+    enterprise-4.15:
```

Comment:
@aireilly, I'm adding the 4.15 telco distro. Please let me know if you want me to also remove the 4.14 one.

Reply:
/lgtm